### PR TITLE
Add LFF features to front end filtering and eligibility questionnaire. 

### LINKED ISSUE #788: [LFF] Add accessibility checkboxes step to eligibility questionnaire
What feature is this part of?
LFF

Is there any context that will help with completing this task?
#609 

Are there any potential sub-steps necessary to complete this task?
- Create a new page in the eligibility questionnaire
- Make sure page is hooked up correctly in the flow
- Add accessibility checkboxes and update the context object

### PR DESCRIPTION
**Pull request by aisiri-murulidhar**
## Issue

- Closes #788 
- Addresses #788
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Adds checkboxes and new page to eligibility questionnaire 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [x] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components
Screen Shot 2021-12-16 at 6 21 20 PM

Screen Shot 2021-12-16 at 6 30 39 PM

### CODE REVIEW

File: sites/public/lib/EligibilityContext.tsx [hunk 1]
```diff
@@ -7,18 +8,71 @@ export enum AgeRangeType {
...
+    rollInShower: boolean,
+    grabBars: boolean,
+    heatingInUnit: boolean,
+    acInUnit: boolean) {
     this.age = age
     this.householdSizeCount = householdSizeCount
     this.income = income
     this.disability = disability
+    this.elevator = elevator
+      this.wheelchairRamp = wheelchairRamp
```
Comment (plunkettgoogle) [inline sites/public/lib/EligibilityContext.tsx:64]: Nit: spacing seems wrong here. But maybe prettier will take care of it?

File: sites/public/pages/eligibility/accessibility.tsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    AppearanceStyleType,
+    Button,
+    FormCard,
+    t,
+    Form,
+    ProgressNav,
+    FieldGroup,
+    Field,
+  } from "@bloom-housing/ui-components"
+  import FormsLayout from "../../layouts/forms"
```
Comment (plunkettgoogle) [inline sites/public/pages/eligibility/accessibility.tsx:15]: Are these spaces at the beginning of the lines right?

File: sites/public/src/forms/filters/FilterForm.tsx [hunk 3]
```diff
@@ -152,6 +152,131 @@ const FilterForm = (props: FilterFormProps) => {
             }}
           />
         </div>
+        <label className="field-label">{t("eligibility.accessibility.title")}</label>
+        <div className="flex flex-row bedroom-selector">
```
Comment (plunkettgoogle) [inline sites/public/src/forms/filters/FilterForm.tsx:156]: Is `bedroom-selector` right?

**Review by plunkettgoogle — COMMENTED** [review]
The checkboxes seem kinda spread out in the eligibility questionnaire, and iono how I feel about the wall of checkboxes in the filter form. It seems like the mocks have the checkboxes aligned vertically, in two columns?

File: sites/public/src/forms/filters/FilterForm.tsx (hunk 3 above)
Comment (aisiri-murulidhar) [reply sites/public/src/forms/filters/FilterForm.tsx]: fixed

File: sites/public/pages/eligibility/accessibility.tsx (hunk 2 above)
Comment (aisiri-murulidhar) [reply sites/public/pages/eligibility/accessibility.tsx]: fixed

File: sites/public/lib/EligibilityContext.tsx (hunk 1 above)
Comment (aisiri-murulidhar) [reply sites/public/lib/EligibilityContext.tsx]: fixed

File: sites/public/styles/overrides.scss [hunk 4]
```diff
@@ -404,6 +404,11 @@ li.progress-nav__item {
   }
 }
 
+.accessibility-filter-selector {
+  column-count: 2;
```
Comment (plunkettgoogle) [inline sites/public/styles/overrides.scss:408]: The checkboxes seem a little spread out, can you snug them up a bit to more closely match the mocks? In both the filter form and the eligibility questionnaire.

File: ui-components/src/locales/general.json [hunk 5]
```diff
@@ -1467,6 +1479,7 @@
         "household": "Household",
         "age": "Age",
         "disability": "Disability",
+        "accessibility": "Accessibility Features",
```
Comment (plunkettgoogle) [inline ui-components/src/locales/general.json:1482]: This makes the progress nav a little cramped, maybe just "Accessibility"?

**Review by plunkettgoogle — APPROVED** [review]

File: ui-components/src/locales/general.json (hunk 5 above)
Comment (aisiri-murulidhar) [reply ui-components/src/locales/general.json:1482]: fixed

File: sites/public/styles/overrides.scss (hunk 4 above)
Comment (aisiri-murulidhar) [reply sites/public/styles/overrides.scss:408]: Fixed!

**Pull request merged**